study-builder/studies/testboston/study-events.conf

```diff
@@ -79,6 +79,22 @@
       "dispatchToHousekeeping": false,
       "order": 3
     },
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": ${id.act.address},
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "ACTIVITY_INSTANCE_CREATION",
+        "activityCode": ${id.act.symptoms_survey}
+      },
+      "cancelExpr": """user.studies["testboston"].forms["SYMPTOMS_SURVEY"].hasInstance()""",
+      "maxOccurrencesPerUser": 1,
+      "delaySeconds" : 259200 #72hrs
+      "dispatchToHousekeeping": true,
+      "order": 4
+    },
```

Comment:
Can you re-organize this? Note that the set of events here are all related to the consent, so having this in the middle here is awkward. Can you put this after this section, with a title `# when completing address` and space it out with newlines?